modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/configurators/TargetPlatformRootProjectConfigurator.java

```diff
@@ -109,15 +111,30 @@ private TargetPlatformRootProjectConfigurator() {
 	private void _addDependenciesTargetPlatformBoms(
 		Project project, String targetPlatformVersion) {
 
+		Matcher dxpVersion = _dxpVersionPattern.matcher(targetPlatformVersion);
+
+		final String bomArtifactId;
+		final String bomCompileOnlyArtifactId;
```

Comment:
If the keyword `final` isn't need please remove it.  Please initialize the variables to keep the code consistent with the code in portal

```
String bomArtifactId = null;
String bomCompileOnlyArtifactId = null;
```